resources/js/components/login/login-form.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class="login__form-wrapper">
+        <form>
+        <p v-if="correctLoginInfo===false" class="form-required m-0">Login failed. Make sure you have the correct access rights.</p>
+        <div class="form-group">
+            <label for="username">CSUN Credentials</label>
+            <p v-if="submitted && !userInfoForm.username" class="form-required m-0">Please enter your credentials</p>
```

Comment:
For a more friendly user experience, please place validation message UNDER the form element instead of above it